src/main/java/taxi/controller/LogoutController.java

```diff
@@ -0,0 +1,16 @@
+package taxi.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class LogoutController extends HttpServlet {
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getSession().invalidate();
+        resp.sendRedirect("/login.jsp");
```

Comment:
Consider using a context-relative URL (e.g. 'resp.sendRedirect(req.getContextPath() + "/login.jsp")') to avoid potential issues with different deployment contexts.